app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb

```diff
@@ -0,0 +1,279 @@
+module Hyrax
+  module Dashboard
+    module CollectionsControllerBehavior
```

Comment:
@jcoyne Can you point me to an example usage in Hyrax?  I searched for 'prepend' but don't see anything that looks equivalent to replacing behavior modules.  The search included hyrax code and the test app.